### PR TITLE
Add downloader service

### PR DESCRIPTION
**Pull request by pixelblend**
See the `README` for more info. Tests are for suckers*.

`* I'll add some once we've proven the concept`

### CODE REVIEW

File: downloader/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "video-dl",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "node --harmony index.js",
```
Comment (andrewn) [inline downloader/package.json]: Is the `--harmony` flag required here?

It's only required if you're using a [flagged item (yellow boxes)](http://node.green/)

File: downloader/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```
Comment (andrewn) [inline downloader/package.json]: Can we change this to `MIT` to match the rest of the repo? Unless you want ISC.

**Comment by andrewn** [comment]
Also, when we come to run this on the Pi we'll need a `systemd` script for it, like this: https://github.com/andrewn/neue-radio/blob/master/systemd/physical.service

File: downloader/package.json (hunk 1 above)
Comment (pixelblend) [reply downloader/package.json]: Probably not then. I thought it would be useful to get the full range of features out of node.

**Pull request merged**